### PR TITLE
add types for `eyevinn-iaf`

### PR DESCRIPTION
**Pull request by oscnord**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by peterblazejewicz** [comment]
what's the relation to:
https://github.com/Eyevinn/ingest-application-framework/blob/main/interfaces/interfaces.d.ts

**Comment by oscnord** [comment]
> what's the relation to:
> https://github.com/Eyevinn/ingest-application-framework/blob/main/interfaces/interfaces.d.ts

https://github.com/Eyevinn/ingest-application-framework is the source project for `eyevinn-iaf`.

**Comment by peterblazejewicz** [comment]
sorry, I'm one of not used to use `eyevinn-iaf`. You have multiple instances of same types already in different places, say here:
https://github.com/Eyevinn/iaf-plugin-aws/blob/main/lib/types/interfaces.ts
so, why do you want to have yet another instance in DT? Do you want to have a single SoT here?

**Comment by oscnord** [comment]
> sorry, I'm one of not used to use `eyevinn-iaf`. You have multiple instances of same types already in different places, say here:
> 
> https://github.com/Eyevinn/iaf-plugin-aws/blob/main/lib/types/interfaces.ts
> 
> so, why do you want to have yet another instance in DT? Do you want to have a single SoT here?

Yes that is correct, we want to have a single SoT here.

File: types/eyevinn-iaf/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+// Type definitions for non-npm package eyevinn-iaf 1.0
+// Project: https://github.com/Eyevinn/ingest-application-framework
+// Definitions by: Eyevinn Technology <https://github.com/Eyevinn>
```
Comment (peterblazejewicz) [inline types/eyevinn-iaf/index.d.ts:3]: this needs to be personal handle, not an  organisation, sorrry!

Comment (oscnord) [reply types/eyevinn-iaf/index.d.ts]: Ok, I've updated it to use personal handles.

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
thx!

**Comment by oscnord** [comment]
Ready to merge

**Pull request merged**